application/pr/viewer.jsx

```diff
@@ -135,6 +151,12 @@ export default class PRViewer extends React.Component {
                                                             </li>
                                                         ;
                                                     }
+                                                    else if (cs[username] === "commitment pending") {
+                                                        return <li key={username}>
+                                                        <a className="bad" href={"https://github.com/" + username +"/"}>{username}</a> needs to submit his non-participant licensing commitment via the link he received by email.
```

Comment:
s/he/they/

Reply:
s/his/their/

Reply:
We'll have that sentence for each contributor so I don't think we should pluralize it.

Reply:
I meant we should they/their as a [neutral singular](https://en.wikipedia.org/wiki/Singular_they)

Reply:
oh, sorry for the confusion. Right, I'll update the code

Reply:
Done: https://github.com/w3c/ash-nazg/pull/163/commits/e8f1735a86a437c7863ceb433e8ac9258d484927